server/mail_templates/ingestDatasetConfirmation.mako

```diff
@@ -51,6 +51,14 @@ Thank you for contributing a ZIP image batch to the ISIC Archive.
     </tbody>
 </table>
 
+% if failedImages:
+    <h4>Failures</h4>
+    <br>One or more files within your dataset failed to be completely imported into the archive:<br>
+    % for failedImage in failedImages:
```

Comment:
If this list is excessively long, we should truncate it. However, that may be too much work for this PR. Can you add a TODO comment and/or file an issue?